src/components/atoms/Input.tsx

```diff
@@ -36,13 +51,26 @@ const Input: React.FC<InputProps> = ({
         input: {
           'aria-label': ariaLabel,
           'aria-labelledby': ariaLabelledBy,
+          endAdornment:
+            type === 'password' && showPasswordToggle ? (
```

Comment:
May not need `type === 'password'`. This gives us the option to hide other sensitive text.

```suggestion
            hideInput ? (
```

Reply:
Good idea!